### PR TITLE
[es] Localize role-based-access-control (RBAC) #2186

### PR DESCRIPTION
**Pull request by franciscorodriguezsv24**
### Describe your changes
Localize the document content/en/role-based-access-control.md to spanish

### Related issue number or link (ex: `resolves #issue-number`)
resolve https://github.com/cncf/glossary/issues/2186

### Checklist before opening this PR (put `x` in the checkboxes)
- [x] This PR does not contain plagiarism
  - don’t copy other people’s work unless you are quoting and contributing it to them.
- [x] I have signed off on all commits 
  - [signing off](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits) (ex: `git commit -s`) is to affirm that commits comply [DCO](https://wiki.linuxfoundation.org/dco). If you are working locally, you could add an alias to your `gitconfig` by running `git config --global alias.ci "commit -s"`.

### CODE REVIEW

File: content/es/role-based-access-control.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
+---
+title: Control de Acceso Basado en Roles (RBAC)
+status: Completed
+category: concepto
```
Comment (electrocucaracha) [inline content/es/role-based-access-control.md:4]:
```suggestion
category: Concepto
```

**Review by ramrodo — CHANGES_REQUESTED** [review]
Mismo comentario de Vïctor. Cambiar a `Concepto`

**Comment by electrocucaracha** [comment]
Parece ser que va a ser necesario agregar [RBAC](https://github.com/cncf/glossary/actions/runs/6092516128/job/16531299895?pr=2354#step:5:31) en el [wordlist ](https://github.com/cncf/glossary/blob/main/content/es/.wordlist.txt) para evitar que el Spellchecker falle

**Comment by franciscorodriguezsv24** [comment]
ok, voy a cerrar este pull request y voy a mandar uno con todos los cambios

**Pull request closed without merging**